routes/web.php

```diff
@@ -21,4 +21,6 @@
     return view('admin.dashboard');
 })->middleware(['auth'])->name('dashboard');
 
+Route::get('profile/{id}',[UserController::class,'profile'])->name('profile');
```

Comment:
As the profile should be only for the 'logged in' user, no need to pass the user id here.

Reply:
done